src/DocSearchModal.tsx

```diff
@@ -317,6 +317,10 @@ export const DocSearchModal: Component<DocSearchModalProps> = ({
                             <a
                               href={hit.url || "#"}
                               aria-label={linkToTheResultAriaLabel}
+                              onClick={(e) => {
+                                // e.preventDefault();
+                                onClose && onClose();
+                              }}
```

Comment:
why was this added?

Reply:
ah so it is related to #176 not #177, feel free to open another PR

Reply:
Thank you for the fix, I've added it in https://github.com/tauri-apps/meilisearch-docsearch/commit/b03fb9484cd39fa5a684bf912ed4b05384ce0793 and attributed you.